test/commandLine/commandLineTest.js

```diff
@@ -0,0 +1,664 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const rimraf = require('rimraf')
+const fork = require('child_process').fork
+
+describe('Command Line Tests', function () {
+  const appDir = path.join(__dirname, '../app/commandLineTest')
+
+  before(function () {
+    fse.ensureDirSync(path.join(appDir))
+    let dataIn = `const path = require('path')
+    const appDir = path.join(__dirname, '../app/commandLineTest')
+    const app = require('../../../roosevelt')({
+      appDir: appDir,
+      generateFolderStructure: false,
+    })
+    if (process.send) {
+      process.send(app.expressApp.get('params'))
+    }
+
+    process.exit()
+    `
+
+    fs.writeFileSync(path.join(appDir, 'app.js'), dataIn)
+  })
+
+  after(function (done) {
+    rimraf(appDir, (err) => {
```

Comment:
we should abstract this into a utility call so that there is one call to `rimraf` across all tests. @Autre31415 down the road we might want to make that utility function *absolutely filled* with defensive logic to ensure it doesn't delete anything that it shouldn't be deleting